### PR TITLE
Update maxVersion string

### PR DESCRIPTION
**Pull request by daxmc99**
Update `maxVersionString`

## Test plan

Review

### CODE REVIEW

**Comment by daxmc99** [comment]
I get an error when running `sg generate`

```
sg generate
👉 Running target "go" (Run go generate [packages...] on the codebase)
ℹ️ go generate ./... (excluding doc/cli/references)
✅ Target "go" done (0s)
❌ go: go generate: 2 errors occurred:
	* exit status 1: panic: error: a migration (1663871069) from a previous version was unexpectedly edited in this release - if this change was intentional add this migration to the allowedOverrideMap  v4.2.0:
up.sql:
  (
  	"""
  	UPDATE changeset_specs
  	SET published = NULL
  	WHERE published = 'null';
+
+ 	UPDATE changeset_specs
+ 	SET published = '"draft"'
+ 	WHERE published = 'draft';

  	ALTER TABLE changeset_specs
  	... // 3 identical lines
  	"""
  )
down.sql:
goroutine 1 [running]:
main.main()
	/Users/dax/work/sourcegraph/internal/database/migration/shared/upgradedata/cmd/generator/main.go:17 +0x70
exit status 2
internal/database/migration/shared/embed.go:9: running "go": exit status 1
	* exit status 1: + export GO111MODULE=on
```

**Review by efritz — APPROVED** [review]

**Comment by efritz** [comment]
@sourcegraph/batchers Can you please check this migration and follow the error instructions to resolve?

```
	* exit status 1: panic: error: a migration (1663871069) from a previous version was unexpectedly edited in this release - if this change was intentional add this migration to the allowedOverrideMap  v4.2.0:
```

File: internal/database/migration/stitch/stitch.go [hunk 1]
```diff
@@ -207,6 +207,7 @@ var allowedOverrideMap = map[int]struct{}{
 	1528395786: {}, // https://github.com/sourcegraph/sourcegraph/pull/18667 - drive-by edit of empty migration
 	1528395701: {}, // https://github.com/sourcegraph/sourcegraph/pull/16203 - rewritten to avoid * in select
 	1528395730: {}, // https://github.com/sourcegraph/sourcegraph/pull/15972 - drops/re-created view to avoid dependencies
+	1663871069: {}, // https://github.com/sourcegraph/sourcegraph/pull/43390 - TODO: @Piszmog to fill in
```
Comment (daxmc99) [inline internal/database/migration/stitch/stitch.go:210]: @sourcegraph/batchers  - I assume the change in https://github.com/sourcegraph/sourcegraph/pull/43390 was intentional. 
Please fill this in with the justification

**Comment by daxmc99** [comment]
@sourcegraph/batchers 
Please use https://github.com/sourcegraph/sourcegraph/pull/45595 to make your changes. Whatever the result is there, I will port over to 4.3.0.



This PR targetted 4.3.0 and I merged to make progress towards an `rc`

**Pull request merged**